components/IntroDiagram.js

```diff
@@ -15,80 +15,91 @@ const IconBox = ({icon, title, content, team}) => (
+                            },
+                        ]}
+                    />
+                    <Box gap="small">
+                        <IconBox 
+                            title="Research"
+                            team="HPE Product Teams"
+                            content="As the first step in the design and development process,
+                                research is incredibly important at HPE. The main sentiment is
+                                'going wide' and then zoning in. This allows for acquiring a large
+                                sample size of what research exists in the world currently and
+                                where HPE wants to take it."
+                            icon={<TestDesktop a11yTitle="research icon" id="research" size="large" />}
+                        />
+                        <IconBox 
+                            title="Design"
+                            team="HPE Product Teams"
+                            content="Once there is a general idea of the best solution, design moves
+                                forward with ideation. This step happens within each HPE product team
+                                to discuss their specific needs with design, as a solution for
+                                HPE Greenlake may differ from HPE Ezmeral. Those with UX/UI specialization 
+                                factor in as well to ensure a seamless experience for the final designs."
+                            icon={<Edit a11yTitle="pencil icon" id="design" size="large" />}
+                        />
+                        <IconBox 
+                            title="Design System"
+                            content="From the team-specific designs, the Design System works to create a 
+                            general design solution, ensuring that anyone at HPE understands how they can 
+                            use these designs in the context of their team."
+                            icon={<System a11yTitle="computer system icon" id="design-system" size="large" />}
+                        />
+                        <IconBox 
+                            title="Development"
+                            content="At this point, development and design come together to discuss
+                                the research information collected at the beginning and the designs
+                                that have been produced as a result. Development has done their own 
+                                research to know the best implementation for the design, adhering 
+                                to the industry's best and latest practices."
+                            icon={<Code a11yTitle="code script icon" id="dev" size="large" />}
+                        />
+                    </Box>
+                </Stack>
             </Box>
-        </Stack>
-    </Box>
-);
+        { size === 'small' && <Box height='small' /> }
```

Comment:
Instead of adding a DOM element just in case window size is small, we should do a better job controlling the existing spacing via pad/margin. This way the `margin` will be calculated better according to the window size.